godtools/App/Features/Menu/Presentation/Menu/MenuViewModel.swift

```diff
@@ -74,24 +72,21 @@ class MenuViewModel: NSObject, MenuViewModelType {
         return ""
     }
     
-    private func setupBinding() {
-        
-        oktaUserAuthentication.didAuthenticateSignal.addObserver(self) { [weak self] (result: Result<OktaAuthUserModel, Error>) in
-            DispatchQueue.main.async { [weak self] in
-                self?.reloadMenuDataSource()
-            }
-        }
+    private func authenticateUser(fromViewController: UIViewController) {
         
-        oktaUserAuthentication.didSignOutSignal.addObserver(self) { [weak self] in
-            DispatchQueue.main.async { [weak self] in
+        authenticateUserUseCase.authenticatePublisher(authType: .attemptToRenewAuthenticationElseAuthenticate(fromViewController: fromViewController))
+            .receiveOnMain()
+            .sink { _ in
+            
+            } receiveValue: { [weak self] _ in
                 self?.reloadMenuDataSource()
             }
-        }
+            .store(in: &cancellables)
     }
     
     private func reloadMenuDataSource() {
         
-        let isAuthorized: Bool = oktaUserAuthentication.isAuthenticated
+        let isAuthorized: Bool = getUserIsAuthenticatedUseCase.getUserIsAuthenticated()
         
         // TODO: Disabling the account section in the menu by hardcoding to false.
```

Comment:
Hey @rachaelblue also note here in ```MenuViewModel``` that the sign in section is hard coded to be disabled.  You'll need to set that in order to test authentication.